bot/exts/halloween/hacktoberstats.py

```diff
@@ -280,14 +285,15 @@ async def get_october_prs(github_username: str) -> Union[List[dict], None]:
             # Ignore logging non-existent users or users we do not have permission to see
             if api_message == GITHUB_NONEXISTENT_USER_MESSAGE:
                 logging.debug(f"No GitHub user found named '{github_username}'")
+                return
             else:
                 logging.error(f"GitHub API request for '{github_username}' failed with message: {api_message}")
-            return
+            return []  # not returning None here, because that should be for when user not found
```

Comment:
```suggestion
            return []  # Not returning None here, because that should be for when user not found
```
Let's capitalize this. I think you should probably focus more on what actually happens as well.

Here is one option. I'm not sure this is better, though. Choose what you think makes the most sense.
```py
return []  # An empty list shows that the user has no valid PRs
```